client/routes/Routes.jsx

```diff
@@ -41,6 +41,10 @@ const Routes = () => {
                 <Route component={Run} />
             </InterceptAnonymizableRoute>
 
+            <InterceptAnonymizableRoute path="/cohort/:cohortId/run/:scenarioId">
+                <Route component={props => <Run {...props} />} />
```

Comment:
Try `render=...`

Reply:
Since I'm working on this in the follow-on branch, I will try this suggestion there.